routes/clients.rb

```diff
@@ -80,15 +80,25 @@ def self.registered(app)
                       contact_email: params[:contact_email] || client.contact_email,
                       contact_name: params[:contact_name] || client.contact_name,
                       active: params[:active] || client.active)
-        redirect "/clients/#{params[:id]}"
-      end
 
-      app.delete '/clients/:id/?' do
```

Comment:
Did you mean to remove the delete route? This code seems to live under app.patch, which is used for editing a client, but probably shouldn't be used to delete one.